solver/job/migration_test.go

```diff
@@ -0,0 +1,104 @@
+package job_test
+
+import (
+	"testing"
+
+	"github.com/omni-network/omni/lib/contracts/solvernet"
+	"github.com/omni-network/omni/lib/tutil"
+	"github.com/omni-network/omni/solver/job"
+	"github.com/omni-network/omni/solver/jobprev"
```

Comment:
old schema temporarily stored in `solver/jobprev`. This test creates an "old" db, then runs the migration.